themes/base.theme.bash

```diff
@@ -464,11 +464,13 @@ function aws_profile {
 }
 
 function safe_append_prompt_command {
-    if [[ -n $1 ]] ; then
-        case $PROMPT_COMMAND in
-            *$1*) ;;
-            "") PROMPT_COMMAND="$1";;
-            *) PROMPT_COMMAND="$1;$PROMPT_COMMAND";;
-        esac
+    local prompt_re="\<${1}\>" # exact match regex
```

Comment:
Yup, I had read it may not be portable and should have made a comment about that in my PR. Unfortunately, I don't have macOS to test with so I was not able to verify. There may be a vagrant box I can use to test with, but I don't think I have the time to look into that right now. So instead I'll have to rely on you for testing on macOS since it seems like you have access to one. :) I'll add a follow up commit to address the macOS issue.